packages/vx-text/src/Text.tsx

```diff
@@ -176,6 +185,11 @@ class Text extends React.Component<TextProps, TextState> {
     const { wordsByLines } = this.state;
     const { x, y } = textProps;
 
+    // Cannot render <text> if x or y is invalid
+    if (!isValidXOrY(x) || !isValidXOrY(y)) {
+      return <svg ref={innerRef} x={dx} y={dy} fontSize={textProps.fontSize} style={SVG_STYLE} />;
```

Comment:
do you think we should return `null` or an `svg`?

Reply:
I was thinking about return null but think if at least having the svg may keep the layout from changing if consumer depends on having some element there.